gradle/libs.versions.toml

```diff
@@ -49,6 +49,7 @@ coroutines = "1.5.2"
 firebase = "28.4.2"
 flexbox = "3.0.0"
 fragmentViewBindingDelegate = "1.0.0"
+hAnalytics = "c2d3db94bc647ef6a8306e78d274525ae628b72f"
```

Comment:
Once `hAnalytics` stabilizes properly, I will move this from being a direct git hash reference to a release-number instead.